client/src/views/Models/components/DrilldownPanel/KnowledgePane.vue

```diff
@@ -10,9 +10,7 @@
         </div>
       </div>
     </div>
-    <div v-else class="alert alert-info" role="alert">
-      No metadata at the moment
-    </div>
+    <message-display v-else :message="`No metadata at the moment.`" />
```

Comment:
I don't think you need the back quote or to bind `message`. A classic attribute should work

Reply:
This has been addressed by replacing it with a slot.